pyproject.toml

```diff
@@ -73,6 +73,8 @@ LINKSTATIC = "1"
 # Standardize the install directory for libraries, as expected by
 # other parts of the wheels build process.
 CMAKE_INSTALL_LIBDIR = "lib"
+#OpenImageIO_BUILD_LOCAL_DEPS = "all"
+IGNORE_HOMEBREWED_DEPS = "1"
```

Comment:
Do you think maybe this is related to inability to find git?

Reply:
Oh snap, I feel like this **must** be it...!

Reply:
Yes, that was it. I'll need to add guardrails around this.